fakes/v0.0/fakes-v0.0.graphql

```diff
@@ -0,0 +1,75 @@
+enum Type {
```

Comment:
Two things: 

1. The enum values should be SCREAMING_SNAKE_CASE
2. Each value should ideally have a description outlining what they are. Also if these are intending to be imported, I'm unsure how users would actually use them in a language without copy/pasting this, as our directives (today) are usually just basic scalars (strings, bools)

Reply:
1. Agreed. Note: is there a GraphQL style guide somewhere as a reference? That'd be useful.
2. Agreed for description, I wanted to hear everyone's opinion before committing to adding description but definitely something that needs to be done.

> I'm unsure how users would actually use them in a language without copy/pasting this, as our directives (today) are usually just basic scalars (strings, bools)